x-pack/plugins/ml/public/components/job_selector/_job_selector.scss

```diff
@@ -0,0 +1,82 @@
+.mlJobSelectorBar {
```

Comment:
Lots of pixel values and hardcoded hex values in this file. Would be nice to clean it up with proper vars.